CMakeLists.txt

```
@@ -54,6 +54,17 @@ if(MSVC)
     add_compile_definitions(WIN32_LEAN_AND_MEAN)
 endif()
 
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "WASI")
```

```suggestion
if(CMAKE_SYSTEM_NAME STREQUAL "WASI")
```
Or is there a reason for writing it like that?